src/EditorFeatures/CSharpTest/Diagnostics/PreferFrameworkType/PreferFrameworkTypeTests_FixAllTests.cs

```diff
@@ -412,9 +412,9 @@ private System.Int32 F(System.Int32 p1, System.Int16 p2)
 using System;
 class ProgramA
 {
-    private int x = 0;
-    private int y = 0;
-    private int z = 0;
+    private Int32 x = 0;
+    private Int32 y = 0;
+    private Int32 z = 0;
```

Comment:
This behavior is what we want. We've consolidated to a single ID, and so now we update all of these when someone does 'fix all'.